packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -907,6 +907,35 @@ Expected response body:
 }
 ```
 
+### Azure_ResourceManager_Models_Resources_ListAvailableOperations
```

Comment:
is this to list which resources are available? or are you looking for available operations on a resource? I'm confused by the name. What about `Azure_ResourceManager_Resources_ListAvailable`

Reply:
It's for listing all available operations for an RP, not for a particular resource or resource type.

Reason for putting it under `Resources`, is that it's a common interface(API) for almost all RPs, and it only has one scenario, which is `interface Operations extends Azure.ResourceManager.Operations {}`. I'm not sure it's worth a separate folder/file.